department-of-veterans-affairs/caseflow
# Added frontend validation for virtual hearing emails

Resolves #13915 

### Description
Adds frontend validation to the Virtual Hearing veteran and Representative Emails

### Acceptance Criteria
- [ ] User is unable to save Hearing Details for Virtual Hearings without a veteran and representative email address

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose a "Virtual" hearing
1. Click the "Edit hearing details" link
1. Delete the veteran email and try to save, confirm that you receive an error message
1. Replace the veteran email address with what it was previously and ensure the save button is disabled
1. Change the veteran email address and ensure that it can be saved
1. Repeat the above steps for the represetative email address to confirm the same behaviour
